docs/licencjat.tex

```diff
@@ -246,6 +246,21 @@ \chapter{Conclusions}
 \chapter{Team members contribution}
 	\krystynaintodo{Podział prac}
 
+\chapter{Dictionary}
+    \begin{itemize}
+        \item Neural network - a machine learning model, inspired by a structure of a human brain. It consits of a large collection of neurons grouped into layers. Each neuron is connected to a set of other neurons. Each connection has an assigned value (also known as weight), which measures its importance.
+
+        \item Training - a process which consists of modification of connection weights after processing input data by a neural network and comparing its output with the desired output.
+
+        \item Inference - a process of computing neural network output based on the input and values of neuron weights
+
+        \item Convolution (discrete) - a mathematical function defined as follows
+        $$ (f\star g)(n) = \sum_{m=-\infty}^{\infty} f[m]*g[n-m]$$ where $$f,g\in Z\rightarrow Z$$
+
+        \item Convolutonal layer - a neural network layer which performs a convolution operation between its inputs and weights (in this case they are usually called weight filters or kernels). Usage of this kind of layers proved to be successful in the field of image recognition, video recognition and natural language processing.
+		\krystynaintodo{dodać więcej}
```

Comment:
wymieniłam te na pewno potrzebne, jak coś się nowego wymyśli to dodamy jak już praca będzie gotowa